### PR TITLE
Update Userlist information

### PR DESCRIPTION
**Pull request by ace0625**
- Update firebase version.
- Fix current status in User list.

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -51,4 +51,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: fdc6d421f2b03ca169b2ed578c9a6e0e5ed756bb
 
-COCOAPODS: 1.1.1
+COCOAPODS: 1.0.1
```
Comment (MechanicKim) [inline ios/Podfile.lock:54]: The version has been downgraded.
Do you guys know the reason?
Is this okay?

Comment (ace0625) [reply ios/Podfile.lock:54]: I think it's okay.

Comment (MechanicKim) [reply ios/Podfile.lock:54]: Yes! I agree with you!

File: src/components/UserList/Row.js [hunk 2]
```diff
@@ -234,12 +238,14 @@ const styles = StyleSheet.create({
   },
   job: {
     fontSize: 14,
-    marginTop: 25,
-    marginLeft: 25,
+    marginTop: 10,
+    marginLeft: CARD_WIDTH * 0.082,
```
Comment (MechanicKim) [inline src/components/UserList/Row.js:242]: '0.082' is used by other lines(242, 243)
We can define a variable and set 0.082.
If this value needs to be changed, We need to change all lines that have '0.082'.

**Comment by MechanicKim** [comment]
I reviewed it.
It is nice to separate animation code into a file.
I know that we will do refactoring later.

Good! Thanks. LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/105/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**